client/js/components/user/DpMasterToebList.vue

```diff
@@ -593,6 +532,67 @@ export default {
       } else if (this.editModeElementId === 0) {
         dplan.notify.error(Translator.trans('error.api.generic'))
       }
+    },
+
+    updateOrga (ident, updatedField, status) {
+      this.updatedItems = this.updatedItems.map(item => {
+        let newItem = item
+        if (item.ident === ident) {
+          newItem = { ...newItem, ...updatedField }
+        }
+        return newItem
```

Comment:
This can be simplified:
```suggestion
        if (item.ident === ident) {
          return { ...newItem, ...updatedField }
        }
        return item
```

Reply:
Done with [c706ee7](https://github.com/demos-europe/demosplan-core/pull/2621/commits/c706ee785dffafea8563a6127954102775662818)